ccmbioinfo/lfs
# LFS-533: Allow upload of vocabulary files

Allows for vocabularies to be uploaded in RDF/XML format, thus allowing for the installation of vocabularies that are not available through BioPortal.


To test:

1. Build the `LFS-533_test` branch
2. Download the **Human Phenotype Ontology** in **RDF/XML** format from https://bioportal.bioontology.org/ontologies/HP
3. Ensure that the `BIOPORTAL_APIKEY` environment variable is **not** set
4. Start LFS
5. Go to *Administration* ... *Vocabularies* and install the downloaded ontology file with the following parameters; `Identifier` = `HP`, `Name` = `Human Phenotype Ontology`, `Version` = `1`
6. Once installation is finished, create a new *Demographics* form.
7. Type something like `myopia` in the co-morbidities field. Auto-complete should happen and answers should allow for selection.
